CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing Guidelines
+
+Hi! Nice to have you with us. This file will walk you through the steps you need to take if you want to contribute a word or some code to the project. Let’s go.
+
+Actually – before we start – a quick word of caution: Unfortunately the contribution process involves some technical knowledge. We aware that this poses a problem for most people and are discussing how a more inclusive contribution process could look like.
+
+## Contributing Words
+
+### Version 1: Issues
+
+If you know a word that should be defined, but – for whatever reason – don’t feel confident writing the definition yourself (or maybe you disagree with an existing defintion), please open an issue. We can then collectively figure out how to best get this word defined.
```

Comment:
I think here we should shift the language to so that contributing new words through Issues is the primary method. Wherever possible, I'd like to remove barriers to entry. 

I prefer if pull requests were a secondary option for those who feel comfortable with it.

Reply:
I’m with you. I just took a go at documenting what’s there and trying to make this as accessible as possible.